webapp/channels/src/components/channel_header/channel_header_title_favorite.tsx

```diff
@@ -34,11 +31,6 @@ const ChannelHeaderTitleFavorite = () => {
         }
     }, [isFavorite, channel?.id]);
 
-    const removeTooltipLink = useCallback(() => {
-        // Bootstrap adds the attr dynamically, removing it to prevent a11y readout
-        toggleFavoriteRef.current?.removeAttribute('aria-describedby');
```

Comment:
`toggleFavoriteRef` since we removed the usage of this ref, we can also remove its defination and not pass it to the button